ubtue_Brill.js

```diff
@@ -70,13 +70,21 @@ function getSearchResults(doc) {
 function postProcess(doc, item) {
 	let title = ZU.xpathText(doc, '//meta[@name="citation_title"]//@content');
 	if (title) item.title = title; 
-	if (!item.abstractNote) {
-	  item.abstractNote = ZU.xpath(doc, '//section[@class="abstract"]//p');
-	  if (item.abstractNote && item.abstractNote.length > 0)
-		 item.abstractNote = item.abstractNote[0].textContent.trim();
-	  else
-		 item.abstractNote = '';
+	let abstracts = ZU.xpath(doc, '//section[@class="abstract"]//p');
+	//multiple abstracts
+	if (abstracts && abstracts.length > 0) {
+		item.abstractNote = abstracts[0].textContent.trim().replace(/^(Résumé\s?)/i, "");
+		if (abstracts.length ) {
```

Comment:
This is always true because of the condition in line 75.

Reply:
ok